frontend/src/features/mapSideBar/tabs/ltsa/LtsaLegalNotationsSubForm.tsx

```diff
@@ -0,0 +1,64 @@
+import { InlineInput } from 'components/common/form/styles';
+import { FieldArray, getIn, useFormikContext } from 'formik';
+import { DescriptionOfLand, LtsaOrders } from 'interfaces/ltsaModels';
+import * as React from 'react';
+import { Col, Row } from 'react-bootstrap';
+import { withNameSpace } from 'utils/formUtils';
+
+export interface ILtsaLegalNotationsSubFormProps {
+  nameSpace?: string;
```

Comment:
This field does not seem to be taking undefined values. Is there a reason for leaving the type like that?

Reply:
These are composable forms. If used in a different context they may require a different, or possibly undefined namespace.

Reply:
But are they meant to be common? given the file location and contents, my guess is that they specific for that. But I could be wrong. Same as the other places where the namespace is passed as possibly undefined

Reply:
Even if they are not common, the order/hierarchy that they are used on this page could change.

Reply:
Think of it this way, a subform has no way of knowing what the parent formik context looks like. As a result, the object that this form operates on could be:
1) at the root level
2) at some child level
3) part of an array
4) etc.

Passing in the an optional namespace reflects the above uncertainty.

Reply:
I see your point, adding that gives some flexibility about how the component could be used in the future. What I am saying is that I have doubts about its future usage, I've seen components that are not reused at all. I prefer the "code for the current intent, not future one" if there is no clear idea of how that would be used in the future. Anyhow, this is a minor thing.